diff --git a/packages/ai/openai/src/Generated.ts b/packages/ai/openai/src/Generated.ts
index 545645962..a45eb1d96 100644
--- a/packages/ai/openai/src/Generated.ts
+++ b/packages/ai/openai/src/Generated.ts
@@ -1314,16 +1314,19 @@ export class CreateChatCompletionResponse extends S.Struct({
     /**
      * Log probability information for the choice.
      */
-    "logprobs": S.NullOr(S.Struct({
-      /**
-       * A list of message content tokens with log probability information.
-       */
-      "content": S.NullOr(S.Array(ChatCompletionTokenLogprob)),
-      /**
-       * A list of message refusal tokens with log probability information.
-       */
-      "refusal": S.NullOr(S.Array(ChatCompletionTokenLogprob))
-    }))
+    "logprobs": S.optionalWith(
+      S.Struct({
+        /**
+         * A list of message content tokens with log probability information.
+         */
+        "content": S.NullOr(S.Array(ChatCompletionTokenLogprob)),
+        /**
+         * A list of message refusal tokens with log probability information.
+         */
+        "refusal": S.NullOr(S.Array(ChatCompletionTokenLogprob))
+      }),
+      { nullable: true }
+    )
   })),
   /**
    * The Unix timestamp (in seconds) of when the chat completion was created.
@@ -3062,7 +3065,7 @@ export class Eval extends S.Struct({
       EvalPythonGrader,
       EvalScoreModelGrader
     )
-  ).pipe(S.propertySignature, S.withConstructorDefault(() => "eval" as const)),
+  ),
   /**
    * The Unix timestamp (in seconds) for when the eval was created.
    */
