diff --git a/packages/ai/openai/src/Generated.ts b/packages/ai/openai/src/Generated.ts
index bff596c3d..72a00600c 100644
--- a/packages/ai/openai/src/Generated.ts
+++ b/packages/ai/openai/src/Generated.ts
@@ -4844,7 +4844,8 @@ export class Includable extends S.Literal(
   "file_search_call.results",
   "message.input_image.image_url",
   "message.output_text.logprobs",
-  "reasoning.encrypted_content"
+  "reasoning.encrypted_content",
+  "web_search_call.action.sources"
 ) {}
 
 export class ListConversationItemsParams extends S.Struct({
