diff --git a/packages/ai/openai/src/Generated.ts b/packages/ai/openai/src/Generated.ts
index bff596c3d..938a26f44 100644
--- a/packages/ai/openai/src/Generated.ts
+++ b/packages/ai/openai/src/Generated.ts
@@ -897,7 +897,7 @@ export class CreateTranscriptionResponseVerboseJson
 {}
 
 export class CreateTranscription200
-  extends S.Union(CreateTranscriptionResponseJson, CreateTranscriptionResponseVerboseJson)
+  extends S.Union(CreateTranscriptionResponseVerboseJson, CreateTranscriptionResponseJson)
 {}
 
 export class CreateTranslationRequestModelEnum extends S.Literal("whisper-1") {}
@@ -4844,7 +4844,8 @@ export class Includable extends S.Literal(
   "file_search_call.results",
   "message.input_image.image_url",
   "message.output_text.logprobs",
-  "reasoning.encrypted_content"
+  "reasoning.encrypted_content",
+  "web_search_call.action.sources"
 ) {}
 
 export class ListConversationItemsParams extends S.Struct({
