diff --git a/packages/ai/openai/src/Generated.ts b/packages/ai/openai/src/Generated.ts
index 1f2580576..a24c4f359 100644
--- a/packages/ai/openai/src/Generated.ts
+++ b/packages/ai/openai/src/Generated.ts
@@ -915,9 +915,9 @@ export class CreateTranscriptionResponseVerboseJson
 {}
 
 export class CreateTranscription200 extends S.Union(
-  CreateTranscriptionResponseJson,
+  CreateTranscriptionResponseVerboseJson,
   CreateTranscriptionResponseDiarizedJson,
-  CreateTranscriptionResponseVerboseJson
+  CreateTranscriptionResponseJson
 ) {}
 
 export class CreateTranslationRequestModelEnum extends S.Literal("whisper-1") {}
