diff --git a/dist/index.mjs b/dist/index.mjs
index 0b8a76abaac754e0a9a5d69ec26e62fc5788aa98..6b8ffafb842aba9b15dde5cc50dc276b6edc3900 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -27,7 +27,14 @@ async function readEnv() {
 }
 //#endregion
 //#region src/dataloader.ts
-const GHDataLoader = new DataLoader(batchLoad, { maxBatchSize: 50 });
+const GHDataLoader = new DataLoader(batchLoad, {
+	maxBatchSize: 50,
+	cacheKeyFn: (request) => JSON.stringify([
+		request.repo,
+		request.kind,
+		request.kind === "pull" ? request.pull : request.commit
+	])
+});
 async function loadCommitData(options) {
 	return await GHDataLoader.load({
 		...options,
@@ -166,7 +173,7 @@ async function getCommitInfo(options) {
 		const bDate = new Date(b.mergedAt);
 		return aDate.getTime() - bDate.getTime();
 	})[0];
-	const author = pr?.author ?? data.author?.user;
+	const author = data.author?.user;
 	return {
 		commit: {
 			sha: options.commit,
@@ -176,7 +183,7 @@ async function getCommitInfo(options) {
 		author: author ? {
 			login: author.login,
 			url: author.url,
-			markdownLink: `[@${author.login}](${author.url})`
+			markdownLink: `@${author.login}`
 		} : void 0,
 		pull: pr ? {
 			number: pr.number,
