diff --git a/dist/changesets-get-github-info.cjs.js b/dist/changesets-get-github-info.cjs.js
index a74df59f8a5988f458a3476087399f5e6dfe4818..ce5e60ef9916eb0cb76ab1e9dd422abcad752bf6 100644
--- a/dist/changesets-get-github-info.cjs.js
+++ b/dist/changesets-get-github-info.cjs.js
@@ -251,18 +251,13 @@ async function getInfo(request) {
     b = new Date(b.mergedAt);
     return a > b ? 1 : a < b ? -1 : 0;
   })[0] : null;
-
-  if (associatedPullRequest) {
-    user = associatedPullRequest.author;
-  }
-
   return {
     user: user ? user.login : null,
     pull: associatedPullRequest ? associatedPullRequest.number : null,
     links: {
       commit: `[\`${request.commit.slice(0, 7)}\`](${data.commitUrl})`,
       pull: associatedPullRequest ? `[#${associatedPullRequest.number}](${associatedPullRequest.url})` : null,
-      user: user ? `[@${user.login}](${user.url})` : null
+      user: user ? `@${user.login}` : null
     }
   };
 }
diff --git a/dist/changesets-get-github-info.esm.js b/dist/changesets-get-github-info.esm.js
index 27e5c972ab1202ff16f5124b471f4bbcc46be2b5..3940a8fe86e10cb46d8ff6436dea1103b1839927 100644
--- a/dist/changesets-get-github-info.esm.js
+++ b/dist/changesets-get-github-info.esm.js
@@ -242,18 +242,13 @@ async function getInfo(request) {
     b = new Date(b.mergedAt);
     return a > b ? 1 : a < b ? -1 : 0;
   })[0] : null;
-
-  if (associatedPullRequest) {
-    user = associatedPullRequest.author;
-  }
-
   return {
     user: user ? user.login : null,
     pull: associatedPullRequest ? associatedPullRequest.number : null,
     links: {
       commit: `[\`${request.commit.slice(0, 7)}\`](${data.commitUrl})`,
       pull: associatedPullRequest ? `[#${associatedPullRequest.number}](${associatedPullRequest.url})` : null,
-      user: user ? `[@${user.login}](${user.url})` : null
+      user: user ? `@${user.login}` : null
     }
   };
 }
