diff --git a/dist/changesets-get-github-info.cjs.js b/dist/changesets-get-github-info.cjs.js
index 4187b5c54532a1c1c179b60cb314026f67e48323..740a894a5f8ac240a5ce1d3f12d2d77cabc0c2aa 100644
--- a/dist/changesets-get-github-info.cjs.js
+++ b/dist/changesets-get-github-info.cjs.js
@@ -237,16 +237,13 @@ async function getInfo(request) {
     b = new Date(b.mergedAt);
     return a > b ? 1 : a < b ? -1 : 0;
   })[0] : null;
-  if (associatedPullRequest) {
-    user = associatedPullRequest.author;
-  }
   return {
     user: user ? user.login : null,
     pull: associatedPullRequest ? associatedPullRequest.number : null,
     links: {
       commit: `[\`${request.commit.slice(0, 7)}\`](${data.commitUrl})`,
       pull: associatedPullRequest ? `[#${associatedPullRequest.number}](${associatedPullRequest.url})` : null,
-      user: user ? `[@${user.login}](${user.url})` : null
+      user: user ? `@${user.login}` : null
     }
   };
 }
diff --git a/dist/changesets-get-github-info.esm.js b/dist/changesets-get-github-info.esm.js
index 071ec75bb2b5894f09c06d4ead395e56ddbfd1c8..9a6134e78ace53ff341ef21f0352f0bd734b6b2c 100644
--- a/dist/changesets-get-github-info.esm.js
+++ b/dist/changesets-get-github-info.esm.js
@@ -228,16 +228,13 @@ async function getInfo(request) {
     b = new Date(b.mergedAt);
     return a > b ? 1 : a < b ? -1 : 0;
   })[0] : null;
-  if (associatedPullRequest) {
-    user = associatedPullRequest.author;
-  }
   return {
     user: user ? user.login : null,
     pull: associatedPullRequest ? associatedPullRequest.number : null,
     links: {
       commit: `[\`${request.commit.slice(0, 7)}\`](${data.commitUrl})`,
       pull: associatedPullRequest ? `[#${associatedPullRequest.number}](${associatedPullRequest.url})` : null,
-      user: user ? `[@${user.login}](${user.url})` : null
+      user: user ? `@${user.login}` : null
     }
   };
 }
