diff --git a/dist/changesets-get-github-info.cjs.js b/dist/changesets-get-github-info.cjs.js
index 4187b5c54532a1c1c179b60cb314026f67e48323..2bd6660da392bca77dab2c8d72e2c3a50f22f5f1 100644
--- a/dist/changesets-get-github-info.cjs.js
+++ b/dist/changesets-get-github-info.cjs.js
@@ -216,6 +216,13 @@ const GHDataLoader = new DataLoader__default["default"](async requests => {
     return cleanedData[repo][data.kind][data.kind === "pull" ? data.pull : data.commit];
   });
-});
+}, {
+  maxBatchSize: 50,
+  cacheKeyFn: request => JSON.stringify([
+    request.repo,
+    request.kind,
+    request.kind === "pull" ? request.pull : request.commit
+  ])
+});
 async function getInfo(request) {
   if (!request.commit) {
     throw new Error("Please pass a commit SHA to getInfo");
@@ -237,16 +237,13 @@ async function getInfo(request) {
     b = new Date(b.mergedAt);
     return a > b ? 1 : a < b ? -1 : 0;
   })[0] : null;
-  if (associatedPullRequest) {
-    user = associatedPullRequest.author;
-  }
   return {
     user: user ? user.login : null,
     pull: associatedPullRequest ? associatedPullRequest.number : null,
     links: {
       commit: `[\`${request.commit.slice(0, 7)}\`](${data.commitUrl})`,
       pull: associatedPullRequest ? `[#${associatedPullRequest.number}](${associatedPullRequest.url})` : null,
-      user: user ? `[@${user.login}](${user.url})` : null
+      user: user ? `@${user.login}` : null
     }
   };
 }
diff --git a/dist/changesets-get-github-info.esm.js b/dist/changesets-get-github-info.esm.js
index 071ec75bb2b5894f09c06d4ead395e56ddbfd1c8..830770b9441e3aa86c1d411d7665a1957998f440 100644
--- a/dist/changesets-get-github-info.esm.js
+++ b/dist/changesets-get-github-info.esm.js
@@ -207,6 +207,13 @@ const GHDataLoader = new DataLoader(async requests => {
     return cleanedData[repo][data.kind][data.kind === "pull" ? data.pull : data.commit];
   });
-});
+}, {
+  maxBatchSize: 50,
+  cacheKeyFn: request => JSON.stringify([
+    request.repo,
+    request.kind,
+    request.kind === "pull" ? request.pull : request.commit
+  ])
+});
 async function getInfo(request) {
   if (!request.commit) {
     throw new Error("Please pass a commit SHA to getInfo");
@@ -228,16 +228,13 @@ async function getInfo(request) {
     b = new Date(b.mergedAt);
     return a > b ? 1 : a < b ? -1 : 0;
   })[0] : null;
-  if (associatedPullRequest) {
-    user = associatedPullRequest.author;
-  }
   return {
     user: user ? user.login : null,
     pull: associatedPullRequest ? associatedPullRequest.number : null,
     links: {
       commit: `[\`${request.commit.slice(0, 7)}\`](${data.commitUrl})`,
       pull: associatedPullRequest ? `[#${associatedPullRequest.number}](${associatedPullRequest.url})` : null,
-      user: user ? `[@${user.login}](${user.url})` : null
+      user: user ? `@${user.login}` : null
     }
   };
 }
