diff --git a/dist/vendor-node.a7c48fe1.js b/dist/vendor-node.a7c48fe1.js
index 6cb555c93482fd24234240de7ee33ce747548361..10886da7fd6842d8f3a1e3283599985a898acc22 100644
--- a/dist/vendor-node.a7c48fe1.js
+++ b/dist/vendor-node.a7c48fe1.js
@@ -7431,6 +7431,8 @@ function createBrowserPool(ctx) {
       off();
     }
   };
+  const Sequencer = ctx.config.sequence.sequencer;
+  const sequencer = new Sequencer(ctx);
   const runTests = async (project, files) => {
     var _a;
     ctx.state.clearFiles(project, files);
@@ -7441,7 +7443,13 @@ function createBrowserPool(ctx) {
     const provider = project.browserProvider;
     providers.add(provider);
     const origin = `http://${((_a = ctx.config.browser.api) == null ? void 0 : _a.host) || "localhost"}:${project.browser.config.server.port}`;
-    const paths = files.map((file) => relative(project.config.root, file));
+
+    let specs = files.map((file) => [project, file]);
+    if (ctx.config.shard)
+      specs = await sequencer.shard(specs);
+    specs = await sequencer.sort(specs);
+    const paths = specs.map(([,file]) => relative(project.config.root, file));
+
     const isolate = project.config.isolate;
     if (isolate) {
       for (const path of paths) {
